regex-assembly/exclude/unix-shell-fps-pl1.ra

```diff
@@ -549,6 +556,10 @@ perms~
 pf
 pf@
 pg
+##! Not an FP at PL 1
+##! php
+php@
+php~
```

Comment:
There was an [issue opened in september](https://github.com/coreruleset/coreruleset/issues/3812) about `php` and space causing false positives, I'm not sure if we should disable `php` at PL-1 or not but that's out of scope for this PR.

Reply:
The way it's now, that issue would be resolved, wouldn't it? We exclude matches of `php `.

Reply:
hm... ok